config/kovan.json

```diff
@@ -430,12 +429,19 @@
           "autoLineTtl": "0",
           "dust": "100",
           "duty": "3",
-          "flipDeploy": {
+          "clipDeploy": {
             "chop": "13",
-            "dunk": "500",
-            "beg": "3",
-            "ttl": "3600",
-            "tau": "3600"
+            "hole": "5000",
+            "chip": "0.1",
+            "tip": "0",
+            "buf": "30",
```

Comment:
It would be more consistent if we specified `buf` as e.g. `1.3`. The pattern here seems to more often be that the only units, not the interpretation, of the values differs from the on-chain representation (with `chop` being an exception that should be corrected in the future, but can be considered out-of-scope for this PR).

Reply:
Changed. You should actually set it to `130`.